content/docs/command-reference/import-url.md

```diff
@@ -131,6 +139,13 @@ $ dvc run -n download_data \
   finish the operation(s)); or if the target data already exist locally and you
   want to "DVCfy" this state of the project (see also `dvc commit`).
 
+- `--to-remote` - Import an external target, but don't move it into the
+  workspace, nor cache it. [Transfer](#example-transfer-to-remote-storage) it
+  directly to remote storage instead. Use `dvc pull` to get the data locally.
```

Comment:
We should probably mention that the default remote is used if set, and otherwise you need to provide -r.

Reply:
Same in add